gfx/display_servers/dispserv_x11.c

```diff
@@ -82,7 +82,7 @@ static bool x11_set_window_decorations(void *data, bool on)
 }
 
 static bool x11_set_resolution(void *data,
-      unsigned width, unsigned height, int f_restore, float hz)
+      unsigned width, unsigned height, int win_hz, float hz)
```

Comment:
What does win_hz mean here? Does it stand for 'window Hz' or 'windows Hz'? If it's the latter, I don't think that is a good name for a variable that we use in files that might not have anything to do with Windows. Just a nitpick but still.